requirements-test.txt

```diff
@@ -0,0 +1,9 @@
+# shpkpr test requirements
+
+mock == 1.3.0
```

Comment:
The [docs](https://pip.pypa.io/en/stable/reference/pip_install/#requirement-specifiers) actually use both styles, so I guess it's down to preference. Would be good to be consistent with the `requirements.txt` file though.